src/I8Beef.UniFi.Video/Protocol/Bootstrap/Bootstrap.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using I8Beef.UniFi.Video.Protocol.Common;
+
+namespace I8Beef.UniFi.Video.Protocol.Bootstrap
+{
+    /// <summary>
+    /// Bootstrap response.
+    /// </summary>
+    public class Bootstrap
+    {
+        /// <summary>
+        /// Admin user group id.
+        /// </summary>
+        public string AdminUserGroupId { get; set; }
+
+        // todo: alertSchedules
+
+        // todo: cameraSchedules
+
+        /// <summary>
+        /// Cameras
+        /// </summary>
+        public IEnumerable<Camera.Camera> Cameras { get; set; }
+
+        /// <summary>
+        /// Cloud host.  Usually video.ubnt.com.
+        /// </summary>
+        public string CloudHost { get; set; }
+
+        // todo: firmwares
```

Comment:
For firmware and maps, can we get the actual objects for these? IList<Map> and IList<Firmware>

The object structures aren't super involved and it'd round these out:

```
maps: [
{
name: "",
mapType: "IMAGE",
lat: 0,
lng: 0,
googleMapType: null,
tilt: 0,
zoom: 0,
_id: ""
},
]
```

firmwares: [
{
platform: "GEN3L",
state: "READY",
path: "",
version: "v4.1.55",
revision: null,
firmwareCode: "UVC.S2L.v4.1.55.f5c44a2.171123.1755",
protocolVersion: 66,
url: null,
md5: "",
source: "BUNDLED",
_id: ""
},
]